### PR TITLE
[Python] Bug fix to robot_driver_provider

### PR DESCRIPTION
**Pull request by qlin960618**
Variable naming inconsistancy and send_joint_limits number of argumenets

### CODE REVIEW

File: src/rosilo_robot_driver/robot_driver_provider.py [hunk 1]
```diff
@@ -59,13 +60,13 @@ def get_target_joint_positions(self):
         return self.target_joint_positions_
 
     def send_joint_positions(self, joint_positions):
-        msg = Float64MultiArray(data=joint_positions)
+        msg = JointState(position=joint_positions)
         self.publisher_joint_states_.publish(msg)
 
-    def send_joint_limits(self, joint_limits):
-        msg_min = Float64MultiArray(data=joint_limits)
+    def send_joint_limits(self, joint_limits_lower, joint_limits_upper):
```
Comment (mmmarinho) [inline src/rosilo_robot_driver/robot_driver_provider.py]: To match the CPP API (https://github.com/rosilo/rosilo_robot_driver/blob/0293563be7805dc688bb07b5b0e676235f41541e/src/rosilo_robot_driver_provider.cpp#L73)
this should be a tuple.

Please change it to something like
```python
def send_joint_limits(self, joint_limits):
  joint_limits_lower, joint_limits_upper = joint_limits
  ...
```

**Review by mmmarinho — CHANGES_REQUESTED** [review]
Almost there, please consider the one comment I made and I'll have this PR accepted.

File: src/rosilo_robot_driver/robot_driver_provider.py (hunk 1 above)
Comment (qlin960618) [reply src/rosilo_robot_driver/robot_driver_provider.py]: I see, I will makethe change

**Review by mmmarinho — APPROVED** [review]
Let me know if there are any more issues later.

**Pull request merged**